index.js

```diff
@@ -0,0 +1,151 @@
+const {Client} = require("pg");
+
+let client;
+
+    if (process.env.NODE_ENV === "test") {
+        client = new Client("postgres://localhost:5432/costume_shop_db_test");
+        client.on("error", (error) => {
+            console.error(error.stack())
+        })
+    } else {
+        client = new Client("postgres://localhost:5432/costume_shop_db_dev");
+        client.on("error", (error) => {
+            console.error(error.stack())
+        })
+    }  
+
+
+const createTables = async () => {
+    await client.query(`
+        DROP TABLE IF EXISTS costumes;    
+    `)
+    await client.query(` 
+        CREATE TABLE costumes(
+            id SERIAL PRIMARY KEY,
+            name VARCHAR(80) NOT NULL,
+            category VARCHAR NOT NULL CHECK(category IN(
+                'adult', 
+                'child', 
+                'baby', 
+                'pet'
+            )), 
+            gender VARCHAR NOT NULL CHECK(gender IN(
+                'male',
+                'female',
+                'unisex'
+            )),
+            size VARCHAR(10) NOT NULL,
+            type VARCHAR(80) NOT NULL,
+            stock_count INTEGER NOT NULL,
+            price FLOAT NOT NULL 
+        );
+    `)
+    // Selecting table created by me, which should just be 'costumes'
+    const table = await client.query(`
+        SELECT * FROM pg_catalog.pg_tables WHERE tableowner='darayazdani';
+    `)
+    console.log(table.rows[0])
+    // Returning the name of the table I created
+    return table.rows[0].tablename;
```

Comment:
This will not work when run on a server, almost certainly. I would not recommend getting the table after creating it. Instead, if you want to test that the table was created, make a query in your test. If you just do `SELECT * FROM customes` it should succeed if the table exists, and would throw an error if it didn't.
```suggestion
```